Source/AviationWeather/Source/utility.h

```diff
@@ -0,0 +1,34 @@
+/**********************************************************************************
+ *                                                                                *
+ * Copyright (c) 2015 Steven Frost, Orion Lyau. All rights reserved.              *
+ *                                                                                *
+ * This source is subject to the MIT License.                                     *
+ * See http://opensource.org/licenses/MIT                                         *
+ *                                                                                *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,    *
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED          *
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.         *
+ *                                                                                *
+ * NOT TO BE USED AS A SOLE SOURCE OF INFORMATION FOR FLIGHT CRITICAL OPERATIONS. *
+ *                                                                                *
+ **********************************************************************************/
+
+#pragma once
+
+namespace aw
+{
+
+//-----------------------------------------------------------------------------
+
+template <class TExpected = double, class TEnum, class TUnit, class TVal, class TLambda>
+bool comparison_conversion_helper(TEnum to, TUnit leftUnit, TUnit rightUnit, TVal const& leftVal, TVal const& rightVal, TLambda && l)
```

Comment:
Was this the thing you wanted to adjust so that we're not doing double equality and instead doing an epsilon delta check?